website/pages/en/tutorials.js

```diff
@@ -182,7 +167,7 @@ class Tutorials extends React.Component {
         length: <translate>2 Hours</translate>,
         prerequisite: true,
         version: "3.0.0",
-        href: "tutorials/upgrade-a-chain/",
+        href: "tutorials/forkless-upgrade/",
```

Comment:
Does this dir and name change break any links elsewhere else the ci might have missed??

Reply:
I did a search and replace all links